pkg/codegen/go/gen_program.go

```diff
@@ -196,7 +196,10 @@ require (
 		}
 
 		// Relatively safe default, this works for Pulumi provider packages:
-		vPath := fmt.Sprintf("/v%d", p.Version.Major)
+		vPath := ""
+		if p.Version != nil && p.Version.Major > 1 {
```

Comment:
Note the `p.Version.Major > 1` part which isn't strictly related. But this seems correct?